apps/docs/content/icons/react-icons/rich-icon-library.mdx

```diff
@@ -13,7 +13,8 @@ All available rich icons in the icon library are shown below.
 
 Use the correct size for each icon. Icons should be used at their original size.
 
-The color of the rich icons can be changed by using the `variant` prop. The following [decorative colors](/tokens/semantic/color#decorative) are available as variants: `option1, option2, option3, option4, option5, option6, option7, option8`.
+The color of the rich icons can be changed by using the `variant` prop. The following [decorative colors](/tokens/semantic/color#decorative) are available as variants: `option1` to `option8`.
+The following status colors are available as well: `success`, `warning`, `danger`, `information` and `upsell`.
```

Comment:
We could link to status colors as well

Reply:
not really... they're not under the status category. It's a bit weird I know...

Reply:
oh alrighty :D